### PR TITLE
Consistent design

### PR DESCRIPTION
**Pull request by TmWgnr**

### CODE REVIEW

File: pages/coffees/[id]/index.js [hunk 1]
```diff
@@ -18,77 +18,130 @@ export default function DetailsPage() {
...
-        </StyledContainer>
-      </ListItem>
-      <Footer></Footer>
-    </>
+      <StyledName>{coffee.name}</StyledName>
+
+      <StyledSection>
+        <StyledCategory>
+          <StyledCategoryLabel>Herkunft</StyledCategoryLabel>
+          {coffee.origins.map((oneOrigin, index) => (
```
Comment (actyvystom) [inline pages/coffees/[id]/index.js:28]: See comments above ⏫

File: pages/coffees/[id]/index.js [hunk 2]
```diff
@@ -18,77 +18,130 @@ export default function DetailsPage() {
...
+            <StyledParagraph key={index}> {oneOrigin}</StyledParagraph>
+          ))}
+        </StyledCategory>
+      </StyledSection>
+      <StyledSection>
+        <StyledCategory>
+          <StyledCategoryLabel>Sorte</StyledCategoryLabel>
+          {coffee.sorts.map((oneSort, index) => (
+            <StyledParagraph key={index}> {oneSort}</StyledParagraph>
```
Comment (actyvystom) [inline pages/coffees/[id]/index.js:37]: See comments above ⏫

File: pages/coffees/[id]/index.js [hunk 3]
```diff
@@ -18,77 +18,130 @@ export default function DetailsPage() {
...
+          ))}
+        </StyledCategory>
+      </StyledSection>
+      <StyledSection>
+        <StyledCategory>
+          <StyledCategoryLabel>Aroma</StyledCategoryLabel>
+
+          {coffee.aroma.map((oneAroma, index) => (
+            <StyledParagraph key={index}> {oneAroma}</StyledParagraph>
```
Comment (actyvystom) [inline pages/coffees/[id]/index.js:46]: See comments above ⏫

**Review by actyvystom — CHANGES_REQUESTED** [review]
Good job, looks nice! 👍🏻
See some comments on inline styles and indices usage in array functions.
Cheers, Thomas

**Review by RonnySteady — APPROVED** [review]

**Review by RonnySteady — APPROVED** [review]
LGTM

**Pull request merged**